src/refdb_fs.c

```diff
@@ -137,27 +141,9 @@ static int packed_reload(refdb_fs_backend *backend)
 	scan = (char *)packedrefs.ptr;
 	eof  = scan + packedrefs.size;
 
-	backend->peeling_mode = PEELING_NONE;
-
-	if (*scan == '#') {
-		static const char *traits_header = "# pack-refs with: ";
```

Comment:
Note the trailing space on this constant. The header usually looks like 
```
# pack-refs with: peeled fully-peeled sorted 
```
Later, when we strstr() for " peeled ", we'll never find it, because that trailing space snips it off.